publishable/database/migrations/2016_01_01_000000_add_voyager_user_fields.php

```diff
@@ -13,7 +13,7 @@ public function up()
             if (!Schema::hasColumn('users', 'avatar')) {
                 $table->string('avatar')->nullable()->after('email')->default('users/default.png');
             }
-            $table->integer('role_id')->nullable()->after('id');
+            $table->unsignedInteger('role_id')->nullable()->after('id');
```

Comment:
We shouldn't be changing existing migrations (except for major version updates) because it won't run for users with existing installs.